ocfweb/api/session_tracking.py

```diff
@@ -106,7 +108,24 @@ def _close_sessions(host):
         )
 
 
+@periodic(60)
+def _cleanup_sessions():
+    """Periodically clean up sessions that don't die naturally.
+
+    For example, if a desktop crashes or is reset.
```

Comment:
I still think this makes a lot more sense as a cronjob, especially since it's more reliable than the ocfweb worker (sometimes we have version mismatches that cause problems) and because this has nothing to do with django, all it does is execute a SQL query. Also, this is already in [ocf_stats/files/stats/bin/close-old-sessions(https://github.com/ocf/puppet/blob/master/modules/ocf_stats/files/stats/bin/close-old-sessions) and is running as a cronjob.

Reply:
I kinda disagree, because keeping the logic here keeps almost everything that has to do with session tracking together, rather than one part living as a solitary cronjob way over in puppet, where it is easily forgotten. Also, in this implementation, even if the worker dies, there won't be any problems with the end times being off by however long the worker was dead, because it just sets `end` to `last_update` after the timeout. Seeing a large number of unclosed sessions would also be an indicator that the worker has died, although, that's a side effect.

Reply:
I do agree having all the logic for this in one place is good, but I still think that using the ocfweb worker as a cron replacement feels wrong, like we're overloading ocfweb to do things that it just doesn't need to be doing, especially since there are other similar cronjobs in ocf_stats for periodically updating MySQL. Either way, we should only have one copy of this, either here, or in puppet, and there are currently 2 copies.

Reply:
I'll remove it from here, it's a smaller change than removing the existing cronjob.